ext/libgav1_android.sh

```diff
@@ -0,0 +1,36 @@
+# This script will build libgav1 for the default ABI targets supported by
```

Comment:
This is consistent with the other ext/.sh files, but I think it would be better to have a #! on an executable file unless this is meant to be run from another script, then maybe it doesn't need to be directly executable.

Reply:
Done.